export.js

```diff
@@ -34,6 +35,8 @@ var paths = {
 	messagesDir: './' + (program.messagesFolder || optionalConfigFile.messagesFolder) ,//'./Client/messages/'
 };
 
+var numOfLangs = (program.numberOfLanguages || optionalConfigFile.numberOfLanguages) || 2;

```

Comment:
These languages are dynamically being found based on the directory.

```
var languages = fs.readdirSync(paths.messagesDir);
// filter out any hidden folders like .DS_STORE
languages = _.filter(languages, function(key) {return key.charAt(0) !== '.'});
```

I would rewrite this line later as 
```
var numOfLangs = (program.numberOfLanguages || optionalConfigFile.numberOfLanguages) || languages.length || 2;
```